packages/domain/pubspec.yaml

```diff
@@ -13,6 +13,7 @@ dependencies:
   meta: ^1.7.0
   get_it: ^7.2.0
   injectable: ^1.5.3
+  json_serializable: ^6.1.5
 
   data:
```

Comment:
very important point - domain should not know anything about data

Reply:
fixed